L/libusb/build_tarballs.jl

```diff
@@ -24,7 +24,8 @@ install_license COPYING
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [p for p in supported_platforms() if !Sys.isfreebsd(p)]
+platforms = [p for p in supported_platforms(;experimental=true) if !Sys.isfreebsd(p)]
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
Wait, no, why expanding the c++ string ABIs here? It doesn't seem to be necessary

Reply:
TBH I am not sure how the cxx string ABIs really work. Looking at the previous failures it seems like it will always be cxx11 ABI? So then the cxx03/cxx11 distinction is not needed?

Reply:
> TBH I am not sure how the cxx string ABIs really work.

See https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions.  In particular note that the string ABI does _**not**_ have to do with the C++ standard (e.g., C++ 11) of the source code.

> So then the cxx03/cxx11 distinction is not needed?

Expanding C++ string ABIs is necessary when the audit says so.  If it doesn't anything, there is no need to expand anything.